frontend/src/app/modules/project/project-settings/project-settings.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Project } from 'src/app/shared/models/project/project';
+import { Component, OnInit} from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { ProjectService } from '@core/services/project.service';
+import { ToastrNotificationsService } from '@core/services/toastr-notifications.service';
+
+@Component({
+  selector: 'app-project-settings',
+  templateUrl: './project-settings.component.html',
+  styleUrls: ['./project-settings.component.sass']
+})
+export class ProjectSettingsComponent implements OnInit {
+  projectId: number;
+  project: Project = {} as Project;
+  tempProjectName: string;
+  tempProjectDescription: string;
+  isLoading = false;
+
+  constructor(
+    private projectService: ProjectService,
+    private toastrService: ToastrNotificationsService,
+    private route: ActivatedRoute
+  )
+  {
+    route.parent.params.subscribe(
+      (params) => this.projectId = params.projectId);
+  }
+
+  ngOnInit(): void {
+    this.getProject(this.projectId);
+  }
+  getProject(projectId: number) {
+      this.isLoading = true;
+      this.projectService
+      .getProjectById(projectId)
+        .subscribe(
+          (resp) => {
+            this.isLoading = false;
+            this.tempProjectName = resp.body.name;
```

Comment:
You don't have to return full response to a component. Let's return only body.

Reply:
its for 'put' and 'post' methods too, or only for 'get'?

Reply:
For all methods